rpc/common/service.proto

```diff
@@ -212,15 +212,37 @@ message DetectedLicense {
   string          name       = 5;
   float           confidence = 6;
   string          link       = 7;
+}
+
+enum LicenseCategory {
```

Comment:
@knqyf263 I have taken the `LicenseCategory` enumeration out of the `DetectedLicense` message. This should not affect backwards compatibility, right?

Reply:
I hope so, but I'm afraid of breaking something. Can you test with an old trivy server and a new trivy client?

Reply:
Tested with Trivy v46.0 server and the scan completed successfully.

Reply:
Thanks!